config.yaml

```diff
@@ -21,3 +21,4 @@ networks:
           - name: Deposit
           - name: Withdrawal
           - name: Transfer
+            logId: "14880471643791846054"
```

Comment:
Since envio version 2.4.1 supports the native Transfer event this solves it for your Custom transfer event